avm/res/azure-stack-hci/logical-network/tests/e2e/defaults/main.test.bicep

```diff
@@ -0,0 +1,203 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-azurestackhci.logicalnetwork-${serviceShort}-rg'
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'ashlnmin'
+
+@description('Optional. A token to inject into the name of each resource. This value can be automatically injected by the CI.')
+param namePrefix string = '#_namePrefix_#'
+
+@description('Optional. The password of the LCM deployment user and local administrator accounts.')
+@secure()
+param localAdminAndDeploymentUserPass string = newGuid()
+
+@description('Required. The app ID of the service principal used for the Azure Stack HCI Resource Bridge deployment.')
+@secure()
+#disable-next-line secure-parameter-default
+param arbDeploymentAppId string = ''
+
+@description('Required. The service principal ID of the service principal used for the Azure Stack HCI Resource Bridge deployment.')
+@secure()
+#disable-next-line secure-parameter-default
+param arbDeploymentSPObjectId string = ''
```

Comment:
Hey @xhy8759, any additional inputs to take not of, that were not already part of the hci-cluster module?